march_hardware_builder/CMakeLists.txt

```diff
@@ -13,6 +13,10 @@ find_package(catkin REQUIRED COMPONENTS
 catkin_package(
     INCLUDE_DIRS include
     CATKIN_DEPENDS
+    march_hardware
+    roscpp
+    roslib
+    urdf
```

Comment:
How come these weren't necessary before?

Reply:
They have always been necessary, but the latest version of catkin_lint detects this problem.